.gitignore

```diff
@@ -3,3 +3,7 @@ workspace
 .idea/
 # build output
 /hive
+
+# don't include target files
+hivesim-rs/target
+simulators/*/target
```

Comment:
nitpick: missed empty line at the  end of the file

Reply:
resolved